### DISCUSSION TITLE
OAuth2 - Missing option for auth url?

### DISCUSSION THREAD

**Discussion by alexl04** [grafana/grafana-infinity-datasource#1036] [Q&A]
Hi, we try to make a connection to our ArcGIS server via OAuth2. We can simply fill in the "client id", "client secret" and "access token url" but we do not have a place holder for the "auth url". Do we miss something or is there a workaround to make it work?

Via Postman the setup works as expected. Without auth url we cannot make it work. Hope someone knows more.

**Comment by yesoreyeram** [comment]
auth url and token url are same.

<img width="675" alt="image" src="https://github.com/user-attachments/assets/aedfcacf-af84-4b54-8e3f-c72defaa2e7f">
